reference/twig_reference.rst

```diff
@@ -637,6 +637,27 @@ file_link
 Generates a link to the provided file and line number using
 a preconfigured scheme.
 
+file_relative
```

Comment:
I think so, but that one already exists for urls.

Reply:
https://symfony.com/doc/current/reference/twig_reference.html#relative-path

Reply:
At least it is consistent with `file_link` function. Maybe should be `|file_relative_path`.

Reply:
I meant, even if there is no name conflict between functions and filters, it could be confusing to have two helpers with the same name and different goals.

Reply:
You are right. Let's keep this name then. Thanks!

Reply:
What about `relative_file_path` instead?